open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/view-generator-connectors/virtualization-solution-connector/src/main/java/org/odpi/openmetadata/openconnectors/governancedaemonconnectors/viewgenerator/virtulaizationsolutionconnector/derby/DerbyConnectorProvider.java

```diff
@@ -0,0 +1,7 @@
+package org.odpi.openmetadata.openconnectors.governancedaemonconnectors.viewgenerator.virtulaizationsolutionconnector.derby;
+
+import org.odpi.openmetadata.frameworks.connectors.ConnectorProviderBase;
+
+public class DerbyConnectorProvider extends ConnectorProviderBase {
```

Comment:
The generic ViewBasedConnectorProvider should extend ConnectorProviderBase and the Derby version should extend ViewBasedConnectorProvider